### PR TITLE
Page/interview-process

### PR DESCRIPTION
**Pull request by RodrigoCastroS**
Creation of a new page for the HR team describing the hiring process. More info on [this XD file](https://xd.adobe.com/view/5dc0eaa5-1df5-476b-8449-6d85b7f77670-0893/screen/bac90064-d824-49f2-bad4-89d678624877/specs/)

### CODE REVIEW

File: version_control/Codurance_September2020/css/pages/interview-overview.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  {{ utils.slate_to_woodsmoke() }}
+}
+
+.video-wrapper h2 {
+  color: white;
+}
+
+
+/*------------------------------------*\
+  #Process interview sections
```
Comment (vicarali) [inline version_control/Codurance_September2020/css/pages/interview-overview.css:113]: For this section, it seems that the image of the text-and-image module is hidden on the small viewpoint of the design (it also was set up that way on the Becoming a Craftsperson page). Should we do it here too?

File: version_control/Codurance_September2020/templates/interview-overview.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    {% module 'interview_process' 
+      path='../modules/Interview Process.module'
+      no_wrapper=True
+    %}
+  </section>
+
+  <section class="video-wrapper" >
+    {% module 'working at codurance' 
+      path='../modules/Embedded Video.module'
+      label="Insinde Codurance"
```
Comment (vicarali) [inline version_control/Codurance_September2020/templates/interview-overview.html:75]: Please don't kill me for this, but it should spell "Inside" :laughing:

**Review by vicarali — APPROVED** [review]
Really great work!! I just left you some comments mainly related to some possible minor spacing improvements; please tell me if you need any clarification or help from my side.

File: version_control/Codurance_September2020/css/pages/interview-overview.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+}
+
+.process-interview .text-and-image__copy h2 {
+  margin-bottom: 1em;
+  font-weight: revert;
+}
+
+{% call utils.large_and_extra_large() %}
+  .process-interview {
+    padding-top: 4em;
```
[the remark this answers is not in the document]
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/css/pages/interview-overview.css:128]: As far as I know, paddings don't add up between each other unless it's in another selector. 2em in a large screems seems too small. We can get into discussing the optical spacing on a design but I don't believe is necessary

File: version_control/Codurance_September2020/css/pages/interview-overview.css [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  }
+{% endcall %}
+
+/*------------------------------------*\
+  #Benefits section
```
[the remark this answers is not in the document]
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/css/pages/interview-overview.css:133]: I'll change that on the module, not here, so we can keep consistency all over

File: version_control/Codurance_September2020/css/pages/interview-overview.css [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+  padding: 4em 0;
+}
+
+/*------------------------------------*\
+  #Recruitment team section
+\*------------------------------------*/
+
+.recruitment-team {
+  padding-top: 4em;
+  padding-bottom: 4em;
```
[the remark this answers is not in the document]
Comment (RodrigoCastroS) [reply version_control/Codurance_September2020/css/pages/interview-overview.css:146]: Same as before about the spacing between sections

File: version_control/Codurance_September2020/css/pages/interview-overview.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (vicarali) [reply version_control/Codurance_September2020/css/pages/interview-overview.css:128]: The 4em in total that I mentioned would indeed come from two different selectors: 2em from process-interview and 2em from text-and-image. But don't worry, if you don't deem it necessary we can leave it as is.

File: version_control/Codurance_September2020/css/pages/interview-overview.css [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+    padding-top: 4em;
+  }
+{% endcall %}
+
+/*------------------------------------*\
+  #Benefits section
+\*------------------------------------*/
+
+.codurance-benefits {
+  padding: 4em 0;
```
[the remark this answers is not in the document]
Comment (vicarali) [reply version_control/Codurance_September2020/css/pages/interview-overview.css:154]: Excuse me, I meant only on the small viewpoint, on the other viewpoints looks great!

**Pull request merged**